pkg/jsonnetfile/jsonnetfile.go

```diff
@@ -33,13 +33,21 @@ var ErrNoFile = errors.New("no jsonnetfile")
 
 // Load reads a jsonnetfile.(lock).json from disk
 func Load(filepath string) (spec.JsonnetFile, error) {
-	m := spec.New()
-
 	bytes, err := ioutil.ReadFile(filepath)
 	if err != nil {
-		return m, err
+		return spec.New(), err
 	}
 
+	return UnmarshalBytes(bytes)
+}
+
+// UnmarshalBytes creates a spec.JsonnetFile from bytes. Empty bytes
+// will create an empty spec.
+func UnmarshalBytes(bytes []byte) (spec.JsonnetFile, error) {
```

Comment:
Very nitpicky: Either `Unmarshal` or `UnmarshaJSON`. The `Bytes` suffix doesn't really include what those bytes are about so either omit it (assume json is common enough) or state that it is JSON, which is always bytes :D

Reply:
Alright.